app/controllers/sessions_controller.rb

```diff
@@ -1,6 +1,7 @@
 class SessionsController < ApplicationController
   include RemoteModal
   allowed_remote_modal_actions :new
+  force_frame_response only: %i[new]
```

Comment:
Why do the allowed_remote_modal_actions and force_frame_response behave different? One can pass keyword arguments directly and the other needs an array?

Reply:
Yes this is a good point I can update the remote modal concern API

Reply:
I have refined a bit the remote modal api. I think it is now clearer 

now we only need to add

```ruby 
 respond_with_remote_modal only: [:new]
```